config/protocol.testnet.neofs.yml

```diff
@@ -24,6 +24,10 @@ ProtocolConfiguration:
   - morph7.t5.fs.neo.org:50333
   VerifyTransactions: true
   P2PSigExtensions: true
+  Hardforks:
+    Aspidochelone: TODO
+    Basilisk: TODO
+    Cockatrice: TODO
```

Comment:
This one needs to be checked. Likely can be from 0 as well.

Reply:
We have no significant (for NeoFS) changes included into this hardfork, and states change doesn't bother us in NeoFS testnet, thus setting it to 0 as well.

Reply:
We need to check compatibility anyway, but Cockatrice is not likely to be a problem, nothing significant changed for NeoFS.